mainframer.sh

```diff
@@ -70,6 +74,27 @@ if [ -z "$REMOTE_COMMAND" ]; then
 	exit 1
 fi
 
+function formatTime {
+    ((h=${1}/3600))
+    ((m=(${1}%3600)/60))
+    ((s=${1}%60))
```

Comment:
Meaningful names are better most of the time, i. e. `hours`, `minutes`, `seconds`.